core/arch/arm/plat-zynq7k/main.c

```diff
@@ -107,6 +107,16 @@ vaddr_t pl310_base(void)
 	return PL310_BASE;
 }
 
+vaddr_t pl310_nsbase(void)
```

Comment:
How about providing this as a common __weak function instead? Since all the other related functions are in `core/arch/arm/kernel/tz_ssvce_pl310_a32.S` I suppose it would be more convenient to implement in assembly in that file.

Reply:
Yes, that would be better